### PR TITLE
Contentful Management API scripts refactor

### PR DESCRIPTION
**Pull request by mendelB**
### What does this PR do?

For the new content management script I was working on, I realized there's a bunch of setups that kinda needs to get repeated. So I figured it'd be nice to set up a general 'Client' that takes care of the basic configuration and fetching the environment client we use for entry updating and then yields to whatever custom script is running.

I hope this setup makes sense and is somewhat clean!

### Any background context you want to provide?
This will pave the way for upcoming PR with another page manipulation script.

### What are the relevant tickets/cards?

[Pivotal #156771576](https://www.pivotaltracker.com/story/show/156771576)

### CODE REVIEW

File: contentful/management-api-scripts/2018_04_17_001_transform_page_slugs.js [hunk 1]
```diff
@@ -1,81 +1,49 @@
...
   console.log(`Updated Page! [ID: ${pageEntry.sys.id}]`);
-  console.log(`--> ${pageEntry.fields.slug[locale]}\n`);
+  console.log(`--> ${pageEntry.fields.slug[LOCALE]}\n`);
 
   // API breather room
   await sleep(1000);
 }
 
-transformPageSlugs();
+contentManagementClient.init(function(environment) {
```
Comment (weerd) [inline contentful/management-api-scripts/2018_04_17_001_transform_page_slugs.js]: Where does `environment` get passed in?

Comment (mendelB) [reply contentful/management-api-scripts/2018_04_17_001_transform_page_slugs.js]: That comes from [here](https://github.com/DoSomething/phoenix-next/pull/851/files#diff-0b148c3293c7c052c64d53008b7fc9beR22)

I thought about it like the migration scripts which pass the `migration` object to the exported functions

File: contentful/management-api-scripts/contentManagementClient.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+const { get } = require('lodash');
+const parseArgs = require('minimist');
+const contentful = require('contentful-management');
+
+const LOCALE = 'en-US';
+
+async function initContentManagementClient(callback) {
+  const args = parseArgs(process.argv, {
+    alias: { 'access-token': 'accessToken', spaceId: 'space-id' },
```
Comment (weerd) [inline contentful/management-api-scripts/contentManagementClient.js]: I might not have caught this in prior PR, but here for `access-token` the key is hyphenated, but in `spaceId` the key is not? I think `access-token` and `accessToken` need to be switched right?

File: contentful/management-api-scripts/2018_04_17_001_transform_page_slugs.js (hunk 1 above)
Comment (weerd) [reply contentful/management-api-scripts/2018_04_17_001_transform_page_slugs.js]: ah cool. ok!

File: contentful/management-api-scripts/contentManagementClient.js (hunk 2 above)
Comment (mendelB) [reply contentful/management-api-scripts/contentManagementClient.js]: Oh LOL totally missed this too!
When I initially set it up, I was switching around the order, and it seems that it doesn't care which way you do it which is cool. But will fix for consistency!

Comment (weerd) [reply contentful/management-api-scripts/contentManagementClient.js]: > it seems that it doesn't care which way you do it which is cool.

😅 haha

**Pull request merged**